src/remediations/fifi.js

```diff
@@ -256,7 +296,7 @@ exports.generatePlaybookRunId = function () {
     return uuidv4();
 };
 
-exports.getConnectionStatus = async function (remediation, account) {
+exports.getConnectionStatus = async function (remediation, account, smart_management) {
```

Comment:
Given that smart_management can be `false` now I think a lot of this code could be optimized (e.g. we probably don't need to fetch sources info if we don't have smart management).